operator/controllers/csiscaleoperator_controller.go

```diff
@@ -2378,11 +2390,59 @@ func validateMaxUnavailableValue(key string, value string, data map[string]strin
 
 func validateHostNetworkValue(inputSlice []string, key, value string, envMap, invalidEnvValue map[string]string) {
 	logger := csiLog.WithName("validateHostNetworkValue")
-        logger.Info("Validating host network input", "inputHostNetwork", value)
-	
+	logger.Info("Validating host network input", "inputHostNetwork", value)
+
 	if containsStringInSlice(inputSlice, strings.ToUpper(value)) {
-		 envMap[key] = value
-	}else{
+		envMap[key] = value
+	} else {
+		invalidEnvValue[key] = value
+	}
+}
+
+// Input can be either of 0.001 OR 1m
```

Comment:
The comment can be improved like below, as we have 20m as lowest limit
// Input must be greater than or equal to 20m